### PR TITLE
Refactor: enforce uniform database naming schema

### LINKED ISSUE #3480: Ensure we have a uniform database-column naming schema through the project
We currently have a mixed schema in different models. They need to be unified such that it's easier to under stand and maintain. 

| Component                       | Naming style (`snake_case`)                                                                                                                                                                                                                                        | Example                                                                                                                                                                                                                                                                                    |
|---------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Table                           | plural                                                                                                                                                                                                                                                             | `posts`, `sessions`, `events`, `speakers`                                                                                                                                                                                                                                                   |
| Model file                      | `<singular of table name>.py`                                                                                                                                                                                                                                      | `event.py`, `session.py`                                                                                                                                                                                                                                                                   |
| Column                          | 1. `<name>` - singular if it represent a single data. Else plural. <br>   2. If date only column: `<name>_date`<br>  3. If date-time column: `<name>s_at` <br> 4. If the column contains an image url, or any type of url, it should be suffixed with `_url` <br> 5. If boolean field: `is_<name>` | 1. `name`, `description` <br> 2. `start_date`, `end_date` <br> 3. `starts_at`, `ends_at` <br> 4. `image_url`, `logo_url`, `event_url`, `ticket_url` <br> 5. `is_active`, `is_admin`, `is_verified`                                                                                                                            |
| Foreign Key id column           | `<singular of table name>_id`                                                                                                                                                                                                                                      | `session_id`, `speaker_id`                                                                                                                                                                                                                                                                 |
| Relationship                    | 1. `<name>` - singular for one-to-one<br> 2. `<name>` - plural for one-to-many<br> 3. `<name>` - singular for many-to-one<br> 3. `<name>` - plural for many-to-many<br>                                                                                          | 1. An event model can have a `call_for_speakers` relation <br> 2. Each track model will have a `sessions` relation <br> 3. Each session model will have a `track` relation <br> 4. Each session model will have a `speakers` relation. Each speaker model will have a `sessions` relation. |
| many-to-many intermediate table | The name is derived from the alphabetical order of the related model names.  `<name_of_model_one>_<name_of_model_two>` (model one and two in alphabetical order)                                                                                                   | A table that holds the relation between events and users will be called.  `event_user`                                                                                                                                                                                                     |

The above table will also be added to the project docs for future reference.

This is a parent issue. Sub-issues can be created.

### PR DESCRIPTION
**Pull request by shubham-padia**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Refactors db according to conventions mentioned in #3480 .

#### Changes proposed in this pull request:

- refer #3480 .

fixes #3480 .

### CODE REVIEW

**Comment by niranjan94** [comment]
@shubham-padia date-time fields seem to be the same ? Was that intentional ? 

Like for examples, in sessions it's still `start_time` and `end_time`. Whereas it should have been `startAt` and `endAt`

**Comment by shubham-padia** [comment]
@niranjan94 That''s left out by mistake, I'm currently busy in writing the blog now, will update as soon as i'm finished.

File: app/static/js/events/wizard/vue-app.js [hunk 1]
```diff
@@ -51,15 +51,15 @@ var app = new Vue({
             return _.sortBy(this.event.tickets, "position");
         },
         invalidDateRange: function () {
-            if(this.event.end_time_date.trim().length <= 0
-                || this.event.end_time_time.trim().length <= 0
-                || this.event.start_time_date.trim().length <= 0
-                || this.event.start_time_time.trim().length <= 0) {
+            if(this.event.ends_at_date.trim().length <= 0
+                || this.event.ends_at_time.trim().length <= 0
```
Comment (shubham-padia) [inline app/static/js/events/wizard/vue-app.js]: Ignore all.

**Comment by shubham-padia** [comment]
@niranjan94 Should be okay now.
And also please don't squash the commits, each has large changes and ought to be seperate.

**Comment by niranjan94** [comment]
@shubham-padia verified. No data loss due to the refactor. ✅

**Pull request merged**